ethereum/scripts/configure-beefy.ts

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+const hre = require("hardhat");
```

Comment:
Should use ES6 `import` instead of `require`.

Reply:
getting annoying typescript errors if I do so, gonna ignore this.